src/main/java/hypersquare/hypersquare/util/PlotUtilities.java

```diff
@@ -101,4 +111,26 @@ public static void moveEntityInsidePlot(Entity entity, Location locA, Location l
             entity.teleport(tpLocation);
         }
     }
+
+    //Stopping blocks indirectly being placed outside plot
+    @EventHandler
```

Comment:
This should not be here, if you want to make a global class for plot event listeners you should do it in another class